### ISSUE TITLE
Can't override OnTeamsMembersRemovedAsync in TeamsActivityHandler 4.11.1

### ISSUE THREAD

**Issue by faceoffers28** [MicrosoftDocs/msteams-docs#2817]
[Enter feedback here]

This is not working in Microsoft.Bot.Builder.Integration.AspNet.Core 4.11.1. I'm getting a "no suitable method found to override" error. The method is clearly listed in the TeamsActivityHandler, though.

```
protected override async Task OnTeamsMembersRemovedAsync(IList<ChannelAccount> membersRemoved, TeamInfo teamInfo, ITurnContext<IConversationUpdateActivity> turnContext, CancellationToken cancellationToken)
        {
            foreach (TeamsChannelAccount member in membersRemoved)
            {
                if (member.Id == turnContext.Activity.Recipient.Id)
                {
                    // The bot was removed
                    // You should clear any cached data you have for this team
                }
                else
                {
                    var heroCard = new HeroCard(text: $"{member.Name} was removed from {teamInfo.Name}");
                    await turnContext.SendActivityAsync(MessageFactory.Attachment(heroCard.ToAttachment()), cancellationToken);
                }
            }
        }

```
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0a6d00a4-dd28-33c4-002e-2e8e7c653ce2
* Version Independent ID: 0856927a-25f4-b1e0-5f1b-d55dc61796f3
* Content: [Subscribe to conversation events - Teams](https://docs.microsoft.com/en-us/microsoftteams/platform/bots/how-to/conversations/subscribe-to-conversation-events?tabs=dotnet#team-members-removed)
* Content Source: [msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md](https://github.com/MicrosoftDocs/msteams-docs/blob/master/msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md)
* Product: **msteams**
* GitHub Login: @WashingtonKayaker
* Microsoft Alias: **anclear**

**Comment by Mallipriya-MSFT** [comment]
Hi @faceoffers28, Thank you for the insights. Will reproduce from our end and update you as soon as possible.

**Comment by faceoffers28** [comment]
@Mallipriya-MSFT I just happened to create another BOT. If you create the Echo Bot Asp.Net Core 3.1 sample, the issue occurs when adding using System.Threading; to enable the CancellationToken. Hope this helps!

**Comment by faceoffers28** [comment]
It looks like a simple documentation error. Here is the correct method. 

       `protected override async Task OnTeamsMembersRemovedAsync(IList<TeamsChannelAccount> teamsMembersRemoved, TeamInfo teamInfo, ITurnContext<IConversationUpdateActivity> turnContext, CancellationToken cancellationToken)
`
Note the “TeamsChannelAccount” and the “teamsMembersRemoved” …… The documentation needs to be updated. I originally noticed the teamsMembersRemoved element, but I didn't see the TeamsChannelAccount object being used.

I replaced the override with this signature and it works. I was able to capture the event when I removed a team member.

Thanks!
